### PR TITLE
Basic structure of the new website

### PR DESCRIPTION
**Pull request by lyvwhyr**
### react + babel + webpack added.
yarn used as package manger `yarn install` to install packages in lock file
webpack dev server can be started with `yarn start` command
sass added for styling, more work to be done but didn't want to make PR to large. 
Original styling left in Main component in src directory

screenshot_2018-10-28_00-09-22

## login state change
screenshot_2018-10-28_00-14-29

## footer
screenshot_2018-10-28_00-12-11

#23

### CODE REVIEW

**Comment by AnshulMalik** [comment]
Nice work @feralCode 

I think we don't need yarn error log

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "website",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/feralCode/website",
```
Comment (AnshulMalik) [inline package.json:5]: we should use the upstream repository here.

File: src/scripts/Main.jsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    return (
+      <div className="container">
+        <main>
+          <div className="intro">
+            <p>
+              We are an
+              <a
+                href="http://en.wikipedia.org/wiki/Open-source_software"
+                target="_blank"
+              >
```
Comment (AnshulMalik) [inline src/scripts/Main.jsx:20]: what's going on here?

File: src/scripts/Main.jsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+                MATLAB /
+                            Octave
+
+              </a>
+
+            </li>
+
+          </ul>
+        </main>
+        <footer>
```
Comment (AnshulMalik) [inline src/scripts/Main.jsx:145]: I think this file needs formatting.

File: src/styles/index.scss [hunk 4]
```diff
@@ -1,18 +1,32 @@
...
+  font-family: Montserrat, Helvetica Neue, sans-serif;
+  margin: 0 auto; line-height: 2em; padding: 12px;
+  color: #333;
+  overflow: auto;
+  background: #fff;
+  width: 100%;
+  padding: 0;
+  margin: 0;
+ //background: #ffded1; /* Old browsers */
+ //background: linear-gradient(135deg,  #ffded1 0%,#ffded1 50%,#ffc5af 50%,#ffc5af 100%); /* W3C */
```
Comment (AnshulMalik) [inline src/styles/index.scss:17]: comments? I think we don't need them.

File: src/scripts/Components/Hero/index.jsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+            {title}
+          </div>
+          <div className="subtitle">
+            {subtitle}
+          </div>
+          <div className="input-container">
+            <input type="text" placeholder={placeholder} autoComplete="off" aria-label="Search box" />
+            <i className="material-icons">search</i>
+          </div>
+          <img className="bg-image" src="https://images.unsplash.com/photo-1527234639945-70d78416bd7d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=721783d6df00ea979608158fd776675e&auto=format&fit=crop&w=1319&q=80" alt="jumbotron background" />
```
Comment (AnshulMalik) [inline src/scripts/Components/Hero/index.jsx:34]: I think we'll be fine without this image for now.

File: src/scripts/Components/Header/index.jsx [hunk 6]
```diff
@@ -0,0 +1,43 @@
+import './index.scss';
+
+import React, { Component } from 'react';
+
+export default class HeaderContainer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      profileImageURL: 'https://d1r8m46oob3o9u.cloudfront.net/images/home-demo-photo-0c.jpg',
```
Comment (AnshulMalik) [inline src/scripts/Components/Header/index.jsx:9]: let's not have any images.

**Review by AnshulMalik — CHANGES_REQUESTED** [review]
So sorry this was ignored for so long, we can now go fast track. :)

**Comment by AnshulMalik** [comment]
@feralCode any updates on this?

**Comment by christianbender** [comment]
@AnshulMalik The last reaction was 27 days ago. Should we close the pull request?

**Pull request closed without merging**